concurrency/2.0/_index.md

```diff
@@ -1,23 +1,23 @@
 ---
-title: "Jakarta Concurrency 2.0 (milestone)"
-date: 2020-05-30
+title: "Jakarta Concurrency 2.0"
+date: 2020-09-12
 summary: "Release for Jakarta EE 9"
 ---
 Jakarta Concurrency provides a specification for using concurrency from application components without compromising container integrity while still preserving the Jakarta EE platform's fundamental benefits.
 
 * [Jakarta Concurrency 2.0 Release Record](https://projects.eclipse.org/projects/ee4j.cu/releases/2.0.0)
   * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
-* [Jakarta Concurrency 2.0 Specification Document](./concurrency-spec-2.0-RC2.pdf) (PDF)
-* [Jakarta Concurrency 2.0 Specification Document](./concurrency-spec-2.0-RC2.html) (HTML)
+* [Jakarta Concurrency 2.0 Specification Document](./concurrency-spec-2.0.pdf) (PDF)
+* [Jakarta Concurrency 2.0 Specification Document](./concurrency-spec-2.0.html) (HTML)
 * [Jakarta Concurrency 2.0 Javadoc](./apidocs)
-* [Jakarta Concurrency 2.0 TCK]()([sig](),[sha](),[pub]())
+* [Jakarta Concurrency 2.0 TCK](https://download.eclipse.org/jakartaee/concurrency/2.0/jakarta-concurrency-tck-2.0.0.zip)([sig](https://download.eclipse.org/jakartaee/concurrency/2.0/jakarta-concurrency-tck-2.0.0.zip.sig),[sha](https://download.eclipse.org/jakartaee/concurrency/2.0/jakarta-concurrency-tck-2.0.0.zip.sha256),[pub](https://raw.githubusercontent.com/jakartaee/specification-committee/master/jakartaee-spec-committee.pub))
 * Maven coordinates
-  * [jakarta.enterprise.concurrent:jakarta.enterprise.concurrent-api:2.0.0-RC2](https://jakarta.oss.sonatype.org/service/local/repo_groups/staging/content/jakarta/enterprise/concurrent/jakarta.enterprise.concurrent-api/2.0.0-RC2/jakarta.enterprise.concurrent-api-2.0.0-RC2.jar)
+  * [jakarta.enterprise.concurrent:jakarta.enterprise.concurrent-api:2.0.0](https://jakarta.oss.sonatype.org/service/local/repo_groups/staging/content/jakarta/enterprise/concurrent/jakarta.enterprise.concurrent-api/2.0.0/jakarta.enterprise.concurrent-api-2.0.0.jar)
 
 
 # Compatible Implementations
 
-* [Jakarta Concurrency CI 2.0.0-RC2](https://jakarta.oss.sonatype.org/service/local/repo_groups/staging/content/org/glassfish/jakarta.enterprise.concurrent/2.0.0-RC2/jakarta.enterprise.concurrent-2.0.0-RC2.jar)
+* [Jakarta Concurrency CI 2.0.0](https://jakarta.oss.sonatype.org/service/local/repo_groups/staging/content/org/glassfish/jakarta.enterprise.concurrent/2.0.0/jakarta.enterprise.concurrent-2.0.0.jar)
 
```

Comment:
CI is different from the compatibility certification request ("https://github.com/eclipse-ee4j/concurrency-api/issues/130")

Reply:
It is concurrent version 2.0.0 that will be included in the final release of GlassFish 6 according to the pom file (https://github.com/eclipse-ee4j/glassfish/blob/master/appserver/pom.xml). I would say that the CI version linked from there should be a link to the released version and not necessarily be the milestone used in the certification request. Should not be a staged version though

Reply:
Not sure to do with the comment? Do we want the version that passed the TCK or do we want the final version incorporated into a to be released GlassFish. It always confuses me these links etc. to things that don't exist yet!

Reply:
Agree, it is confusing. Here's my take:

- The link in the _index.md file should be to the download location of the final release of the compatible implementation. Could be a link to something that is not there yet
- The link in the certification request should be to the download location of the specific version (milestone, release candidate, etc.) that was used to pass the TCK. It should not be a snapshot, or point to a location or file that will be removed later. It should be persistent.

Reply:
So we should be releasing RCs to maven central?

Reply:
If you're certifying on an RC, I think you should. Unless you can provide a separate download location for it that does not go away.

Reply:
Did you use Glassfish 6.0-M1 to do the compatibility testing for Concurrency 2.0?  If so, that is a permanent link.

Reply:
yes I did use GlassFish M1